lib/grab-module-data.js

```diff
@@ -48,7 +48,9 @@ function grabModuleData(context) {
         return;
       }
 
-      if (data === '') {
+      // Prior to npm 7, returns empty string for invalid request
+      // In npm 7 newline is returned for invalid request
+      if (data === '' || data === '\n') {
```

Comment:
Are we sure it doesn't use CRLF on Windows? We could otherwise check for `data.trim() === '')`